unit-tests/unit-tests-post-processing.cpp

```diff
@@ -237,6 +258,7 @@ TEST_CASE("Post-Processing Filters sequence validation", "[software-device][post
 
                 // Compare the resulted frame versus input
                 validate_ppf_results(depth, filtered_depth, test_cfg, i);
+                compare_frame_md(depth, filtered_depth);
```

Comment:
We will need more recording as the currently existing records do not include metadata